### PR TITLE
Adds EmptyState, Horizontal Pagination and Artwork Counts to Partner also adds isCloseToEdge util

### PR DESCRIPTION
**Pull request by kierangillen**
- Adds `EmptyState` to each scrollview
- Creates a horizontal pagination container for current and upcoming shows
- Creates a `isCloseToEdge` method for horizontal pagination
- Adds artworks count to header

## Empty state
Screen Shot 2019-11-05 at 12 00 52 PM

#skip_new_tests

### CODE REVIEW

File: src/lib/utils/isCloseToEdge.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ *      <Artworks />
+ *    </Flatlist>
+ *  )
+ */
+
+const PAGE_END_THRESHOLD = 1000
+
+type CallBack = () => void
+
+export function isCloseToEdge(onScrollEnd: CallBack, pageEndThreshold: number = PAGE_END_THRESHOLD) {
```
Comment (ashfurrow) [inline src/lib/utils/isCloseToEdge.ts]: Are we planning to add tests for this class? I'm comfortable merging as-is as long as we ticket the follow-up tests.

Comment (kierangillen) [reply src/lib/utils/isCloseToEdge.ts]: We can, looks like there are no tests yet for `isCloseToBottom` so they could be grouped together.

Comment (kierangillen) [reply src/lib/utils/isCloseToEdge.ts]: Added a ticket: https://artsyproduct.atlassian.net/browse/ME-99

**Comment by ashfurrow** [comment]
The tests have passed but GitHub hasn't updated the build status. Merging.

**Comment by artsyit** [comment]
:rocket: PR was released in v1.18.22 :rocket:

**Pull request merged**